### PR TITLE
[FIX] account_analytic_default: fix ocr not using default analytic account

### PR DESCRIPTION
**Pull request by sdegueldre**
PURPOSE:

Default analytic account should be used wherever possible. This wasn't
the case because the module didn't implement a default_get and only
filled the fields on product_id change, which isn't set by the OCR

task-2067097
opw-2068110

### CODE REVIEW

File: addons/account_analytic_default/models/account_analytic_default.py [hunk 1]
```diff
@@ -63,6 +63,27 @@ def account_get(self, product_id=None, partner_id=None, user_id=None, date=None,
 class AccountInvoiceLine(models.Model):
     _inherit = "account.invoice.line"
 
+    def default_get(self, fields_list):
```
Comment (mreficent) [inline addons/account_analytic_default/models/account_analytic_default.py]:
```suggestion
   @api.model
    def default_get(self, fields_list):
```

Comment (sdegueldre) [reply addons/account_analytic_default/models/account_analytic_default.py]: whoops! fixed in 2c0a2b2bea5de1d3f330dbb4ec18057280bb33d1

**Comment by nim-odoo** [comment]
ping @fda-odoo

**Comment by fda-odoo** [comment]
robodoo r+

**Pull request closed without merging**